articles/quickstart/spa/angular-next/02-calling-an-api.md

```diff
@@ -0,0 +1,186 @@
+
+The focus of this guide is to show you how to configure the SDK to call APIs that require an access token. Instead of creating a demo API to test the client-server connection, you'll use the [Auth0 Management API](https://auth0.com/docs/api/management/v2), which comes bundled with your Auth0 tenant. However, you can adapt this guide to work with any API that you are securing with Auth0.
+
+:::note
+If you followed the [previous section where you added user log in to Angular](/quickstart/spa/angular-next#add-login-to-your-application), make sure that you log out of your application as you'll need a new access token to call APIs.
+:::
+
+## Configure Your App to Make Authorized API Calls
+
+To facilitate calling APIs, the Angular SDK exports an [`HttpInterceptor`](https://angular.io/api/common/http/HttpInterceptor) that automatically adds an `Authorization` header with the access token to outgoing requests. However, only requests that are specified through configuration will have this header attached, preventing unintended leakage of access tokens to unexpected URLs.
+
+To make use of the HTTP interceptor, follow these steps:
+
+* Import the `AuthHttpInterceptor` type from the Auth0 Angular SDK
+* Import `HttpClientModule` and `HTTP_INTERCEPTOR` from `@angular/common/http`
+* Register `AuthHttpInterceptor` in the `providers` section of your application module
+* Add configuration to specify which requests should have an `Authorization` header
+
+Once the above steps have been completed, any calls to APIs through Angular's `HttpClient` class to configured URLs will have an `Authorization` header attached.
+
+The following section demonstrates how to configure the application module to call APIs.
+
+### Configure the Auth Module
+
+The `AuthModule` setup is similar to the one discussed in the [Configure the `Auth0Provider` component](/quickstart/spa/angular-next##register-and-configure-the-authentication-module) section: you import the `AuthModule` type from the SDK into your Angular application module and call the static `forRoot` method, passing the `domain` and `clientId` properties. The values of these two props come from the ["Settings" values](https://auth0.com/docs/quickstart/spa/angular-next#configure-auth0) of the single-page application you've registered with Auth0.
+
+However, your Angular application needs to pass an access token when it calls a target API to access private resources. You can [request an access token](https://auth0.com/docs/tokens/guides/get-access-tokens) in a format that the API can verify by passing the `audience` and `scope` props to `AuthModule.forRoot`.
+
+In addition, the configuration should specify which requests should have an `Authorization` header added to them with the access token.
+
+The following example shows how to set up your Angular application module, with all the necessary imported types and the configuration to call the management API:
```

Comment:
Does this overrides stuff that the reader might have written in the former chapter? Is there an alternative path in whihc this code is added to a clean starting point? In both cases, it seems the instructions should be more specific